zuul.d/jobs.yaml

```diff
@@ -33,7 +33,7 @@
         - name: focal-medium
           label: focal-medium
     secrets: *log_clouds
-    branches: ^stable\/(1\.5|1\.6|1\.7|1\.8|4\.0|4\.1|4\.2|20\.03|20\.12|21\.09|22\.03|22\.09|21\.10|focal|jammy|train|ussuri|victoria|wallaby|xena|yoga|zed|luminous|mimic|nautilus|octopus|pacific|quincy|quincy\.2).*$
+    branches: ^stable\/(1\.5|1\.6|1\.7|1\.8|4\.0|4\.1|4\.2|20\.03|20\.12|21\.09|22\.03|22\.09|21\.10|focal|jammy|train|ussuri|victoria|wallaby|xena|yoga|zed|luminous|mimic|nautilus|octopus|pacific|quincy|quincy\.2)$
```

Comment:
we should use a list of branches like you did in the project-templates.yaml change.

Reply:
Yes, I agree. Let me actually pull this change out into it's own change because it is not necessarily related.